intellij/README.md

```diff
@@ -0,0 +1,8 @@
+# IntelliJ Configuration Files 
+
+Here you can find .xml for use with intelliJ, which you can import from `Preferences -> Editor -> Code Style -> Java`
```

Comment:
Maybe we should point out that this is best effort, should be considered work in progress, and may not exactly represent the checkstyle requirements, in order to encourage people to open PRs to improve it?

Reply:
Yeah, supposedly the CH code repos are different, I imagine the Falcon ones are too, so adding a few more files // improving this would be good.

Reply:
```suggestion
Here you can find .xml for use with IntelliJ, which you can import from `Preferences -> Editor -> Code Style -> Java`.
```